### DISCUSSION TITLE
[blocks]: Filesystem dashboard

### DISCUSSION THREAD

**Discussion by andelwebsolutions** [shadcn-ui/ui#3168] [Blocks Request]
### Description

A Google Drive-like Dashboard where you can upload files and manage folders would be awesome in the shadcn-ui style!

### Example

https://x.com/imuratalpay/status/1732766602687349166?s=46

**Comment by ruymon** [comment]
Other good example: https://geist-ui.dev/en-us/components/file-tree

**Comment by birobirobiro** [comment]
Another examples: 

* DaisyUI: https://daisyui.com/components/menu/#file-tree
* Nextra: https://nextra.site/docs/guide/built-ins/filetree
* https://github.com/shadcn-ui/ui/issues/355#issuecomment-1703767574

**Reply by ruymon** [reply]
Biro!!! Brabo! 🇧🇷
